recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb

```diff
@@ -1,167 +1,164 @@
-  install -d ${D}${sysconfdir}/omnect/consent
-  install -m 0660 -o adu -g adu ${S}/src/extensions/step_handlers/swupdate_consent_handler/files/consent_conf.json ${D}${sysconfdir}/omnect/consent/
-  install -m 0660 -o adu -g adu ${S}/src/extensions/step_handlers/swupdate_consent_handler/files/history_consent.json ${D}${sysconfdir}/omnect/consent/
-  install -m 0660 -o adu -g adu ${S}/src/extensions/step_handlers/swupdate_consent_handler/files/request_consent.json ${D}${sysconfdir}/omnect/consent/
-  install -d ${D}${sysconfdir}/omnect/consent/swupdate
-  install -m 0660 -o adu -g adu ${S}/src/extensions/step_handlers/swupdate_consent_handler/files/user_consent.json ${D}${sysconfdir}/omnect/consent/swupdate/
-  install -m 0660 -o adu -g adu /dev/null ${D}${sysconfdir}/omnect/consent/swupdate/installed_criteria
-  echo "consent ${SOFTWARE_VERSION}" >> ${D}${sysconfdir}/omnect/consent/swupdate/installed_criteria
-
-  # delete adu-swupdate.sh
-  rm ${D}${libdir}/adu/adu-swupdate.sh
-}
-
-do_install:append:omnect_grub() {
-  # install the swupdate_v2 script only for devel images
-  if ${@bb.utils.contains('OMNECT_RELEASE_IMAGE', '0', 'true', 'false', d)}; then
-    install -d ${D}${libdir}/swupdate
-    install -m 0755 ${WORKDIR}/swupdate_handler_v2_grub.sh ${D}${libdir}/swupdate/omnect-swupdate.sh
-  fi
-}
-
-do_install:append:omnect_uboot() {
-  # install the swupdate_v2 script only for devel images
-  if ${@bb.utils.contains('OMNECT_RELEASE_IMAGE', '0', 'true', 'false', d)}; then
-    install -d ${D}${libdir}/swupdate
-    install -m 0755 ${WORKDIR}/swupdate_handler_v2_u-boot.sh ${D}${libdir}/swupdate/omnect-swupdate.sh
-  fi
-}
-
-pkg_postinst:${PN}() {
-  sed -i "s/@@UID@@/$(id -u adu)/" $D${sysconfdir}/aziot/keyd/config.d/iot-hub-device-update.toml
-  sed -i -e "s/@@UID@@/$(id -u adu)/" -e "s/@@NAME@@/AducIotAgent/" $D${sysconfdir}/aziot/identityd/config.d/iot-hub-device-update.toml
-}
-
-SYSTEMD_SERVICE:${PN} = " \
-  deviceupdate-agent.service \
-  deviceupdate-agent.timer \
-"
-
-FILES:${PN} += " \
-  ${@bb.utils.contains('OMNECT_RELEASE_IMAGE', '1', '', '${libdir}/swupdate/omnect-swupdate.sh', d)} \
-  ${libdir}/adu \
-  ${libdir}/tmpfiles.d/iot-hub-device-update.conf \
-  ${sysconfdir}/aziot/keyd/config.d/iot-hub-device-update.toml \
-  ${sysconfdir}/aziot/identityd/config.d/iot-hub-device-update.toml \
-  ${systemd_system_unitdir}/deviceupdate-agent.service \
-  ${systemd_system_unitdir}/deviceupdate-agent.timer \
-  ${sysconfdir}/omnect/consent/consent_conf.json \
-  ${sysconfdir}/omnect/consent/history_consent.json \
-  ${sysconfdir}/omnect/consent/request_consent.json \
-  ${sysconfdir}/omnect/consent/swupdate/user_consent.json \
-  ${sysconfdir}/omnect/consent/swupdate/installed_criteria \
-  "
-
-GROUPADD_PARAM:${PN} += " \
-  -r adu; \
-  -r do; \
-"
-USERADD_PARAM:${PN} += "--no-create-home -r -s /bin/false -G aziotcs,aziotid,aziotks,do -g adu adu;"
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:${LAYERDIR_omnect}/files:"
```

Comment:
why is the whole file a diff?

Reply:
autoremove of line endings

Reply:
please revert that

Reply:
turned back to original line ending cr+lf